### DISCUSSION TITLE
Process community rewards cycle: 1/14/23 - 2/10/23

### DISCUSSION THREAD

**Discussion by AlgoRhythMatic** [asastats/channel#468] [Rewards]
Another cycle has passed, so it is once again time to distribute rewards to our community contributors based on our newly refined measurements listed in the bottom of the page of our [Ongoing Contributions spreadsheet](https://docs.google.com/spreadsheets/d/1UJ9ZyvV0fdcXAfpUNOsRa1aCmJko-QhF-BU4Wcu6Xog/edit#gid=238932857):

```
# NOTE ASA Stats Community rewards 1/14/23 - 2/10/23
@alexander_lakis 0.03 damo
@JimCoinASA 0.03 damo
@RaccoonSquadNFT 0.03 damo
AlgoRhythMatic 0.15 damo
Algoworm 0.12 damo
BartleDooDerp  0.18 damo
DK+-x 0.03 damo
DragMZ 0.06 damo
kerrilija 0.65 damo
Milesmile 0.03 damo
moloch10 0.36 damo
motuwagon 0.03 damo
q2ev 0.03 damo
sensorium  0.07 damo
t@Miki 0.06 damo
UncleDooom 0.03 damo
TOTAL 1.89 damo
```
Please review and comment if there are any questions or concerns. The rewards noted above for this period use newly-refined rewards values discussed here: https://github.com/asastats/channel/discussions/405

Per usual process, we'll wait ~24 hrs for comments/feedback, and then pass on the list to the rewards sender (currently @ipaleka).

As well, please comment or reach out to me if you are interested in participating in rewards measurements in future cycles!

**Comment by AlgoRhythMatic** [comment]
@ipaleka - @kerrilija noted that t@miki = Algoworm.  In cases like this, where a community member has 2 or more usernames across multiple platforms, does your processing breakup the rewards into multiple transactions, or do the transactions combine per-address?

I'm happy to combine or update if/when we know that there are multiple usernames vs. where the request comes from - but just trying to get a better understanding of how best to simplify.

**Reply by ipaleka** [reply]
In this case, I just added the same address to the user Algoworm, so DragMZ script is able to allocate funds.

I guess the easiest way is that you just use the username you find in the issue and if you somehow know that user's previous username you can make a notice either here or in my DM, so I don't need to reach them asking for an address.

**Comment by ipaleka** [comment]
[This issue](https://github.com/asastats/channel/issues/463) has been pretty complicated, so I checked how you did it. It looks like kerrilija misses his reward for it.

I named it [F2] and in the body of the issue there are two sections. The original poster is krerrilija so he definitely gets the IC for it, but the thing is that he's also a reporter of the issue, tho he hadn't been aware that RaccoonSquadNFT also reported the same.

So I just added a horizontal separator to the same body and c/p the second reporter there, and finally set that [F2] to be divided between those two, plus, of course, IC to kerrilija.

Do you think I should use some other technique to emphasize recipients in such a case?

And to resolve this, I guess you just need to add another same row in the spreadsheet for kerrilija just above RaccoonSquadNFT's.

**Reply by AlgoRhythMatic** [reply]
So I think there were then two mistakes on my behalf:
<img width="1034" alt="Screenshot 2023-02-12 at 10 17 56" src="https://user-images.githubusercontent.com/97758273/218329446-cc66d61b-0057-4ce3-89ba-5fecfc7da769.png">

1) Understanding that @kerrilija get's 50% of [F2]
2) Actually classifying as [F2].

Looks like I had @kerrilija marked correctly as [IC1], and then @RaccoonSquadNFT as [F1].  I can add an extra line and add both as [F2] at 50% each if this is for sure the intent.

Edit: I've added the extra line to spreadsheet + updated both of the feature entries to [F2] x 50%.  Here is the updated output:

```
# NOTE ASA Stats Community rewards 1/14/23 - 2/10/23
@alexander_lakis 0.03 damo
@JimCoinASA 0.03 damo
@RaccoonSquadNFT 0.03 damo
AlgoRhythMatic 0.15 damo
Algoworm 0.12 damo
BartleDooDerp  0.18 damo
DK+-x 0.03 damo
DragMZ 0.06 damo
kerrilija 0.68 damo
Milesmile 0.03 damo
moloch10 0.36 damo
motuwagon 0.03 damo
q2ev 0.03 damo
sensorium  0.07 damo
t@Miki 0.06 damo
UncleDooom 0.03 damo
TOTAL 1.92 damo
```

Thank you for fact-checking!

**Reply by ipaleka** [reply]
That's it then! :)

Do you have a hint for me on how I should emphasize differently that there are two recipients? Besides the horizontal line like this:

- - - - -

Is there some other way I should use?

**Reply by AlgoRhythMatic** [reply]
Yes, I think horizontal line will work!

**Reply by ipaleka** [reply]
> Yes, I think horizontal line will work!

Argh, I just spotted a non-resolved issue marked as simple and easy to grasp [F1+F1]. Sorry I made such confusion with that last one, in the future whenever two (or three...) contributors share the rewards then I'll mark them as [F1+F1] **and** add a horizontal line between them.

Sorry again!

**Reply by AlgoRhythMatic** [reply]
Ah yes - I thought you had done it this way in the past, and it was also easy to understand from title 😀 But of course this requires that you potentially update a title it if it was created by someone else who didn’t credit themself (though I guess so does adding divider potentially).

**Reply by ipaleka** [reply]
That's ok, no problem at all! I totally forgot about plus sign which is clear, simple, and straightforward - a horizontal line is there just aesthetic-wise. :)

**Comment by moloch10** [comment]
Thank you all. 🤗
